.github/workflows/verify.yaml

```diff
@@ -114,4 +114,36 @@ jobs:
           rm log.txt
           killall java      
 
-      # we cannot check sample 5 currently because we'd need repo secrets for that (client id,...)
\ No newline at end of file
+      # we cannot check sample 5 currently because we'd need repo secrets for that (client id,...)
+
+
+  Helm-Chart:
+    runs-on: ubuntu-latest
+    name: Test Helm chart in minikube
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+          cache: 'gradle'
+
+      - name: Start minikube
+        uses: medyagh/setup-minikube@master
```

Comment:
This project does not seem too popular. How risky is it to base a CI pipeline on that?

Reply:
Replaced with shell script